src/main/java/org/gridsuite/study/server/repository/TemporaryStudyRepository.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.repository;
+
+import org.springframework.data.cassandra.repository.Query;
+import org.springframework.data.cassandra.repository.ReactiveCassandraRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Mono;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+
+@Repository("temporarystudy")
+public interface TemporaryStudyRepository extends ReactiveCassandraRepository<TemporaryStudy, String> {
+
+    @Query("SELECT * FROM temporaryStudy WHERE studyName = :studyName")
```

Comment:
Do you really need to specify the query. Isn't it automatically generated from method name?

Reply:
Done